src/storage/txn/commands/prewrite.rs

```diff
@@ -1074,25 +1075,28 @@ mod tests {
         ];
         let mut statistics = Statistics::default();
         // calculated_ts > max_commit_ts
-        let cmd = super::Prewrite::new(
-            mutations,
-            k1.to_vec(),
-            20.into(),
-            0,
-            false,
-            2,
-            TimeStamp::default(),
-            40.into(),
-            Some(vec![k2.to_vec()]),
-            false,
-            Context::default(),
-        );
-
-        let res = prewrite_command(&engine, cm, &mut statistics, cmd).unwrap();
-        assert!(res.min_commit_ts.is_zero());
-        assert!(res.one_pc_commit_ts.is_zero());
-        assert!(!must_locked(&engine, k1, 20).use_async_commit);
-        assert!(!must_locked(&engine, k2, 20).use_async_commit);
+        // Test the idempotency of prewrite when falling back to 2PC.
```

Comment:
How about the case that, for a prewrite request with `k1` and `k2`, the `max_ts` calculated for `k1` is ok, while the `max_ts` of the concurrency manager changed before the process of `k2` and the prewrite of `k2` fallbacked to 2pc.
Then if the prewrite request succeded and the request is retried, it's needed to verify the `min_commit_ts` in the response is zero.